pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py

```diff
@@ -60,6 +60,8 @@
     "'123456789' = 123 terabytes and a bit less than a half\n"
 )
 
+DESIRED_CAPACITY_ERROR = DESIRED_CAPACITY_HINT
```

Comment:
I think the other usage is mine, and I'm happy with the rename, if that helps.

Reply:
I was considering it, but `DESIRED_CAPACITY_ERROR` makes more sense in the context of the validation error. It might be useful to have both constants.